src/dialect/database-introspector.ts

```diff
@@ -68,4 +68,5 @@ export interface ColumnMetadata {
   readonly isAutoIncrementing: boolean
   readonly isNullable: boolean
   readonly hasDefaultValue: boolean
+  readonly comment?: string | null
```

Comment:
Optional is enough. No need to make this also nullable. What would the difference between `undefined` and `null` value be?